src/daemon/daemon.go

```diff
@@ -38,9 +38,9 @@ func (b *BademeisterDaemon) processTransaction(tx *types.Transaction) error {
 }
 
 func (b *BademeisterDaemon) processBlock(block *types.Block) error {
-	log.Printf("Received block, updating transactions")
-	// TODO update storage
-	return nil
+	log.Printf("Received block, updating database")
```

Comment:
nit: could log block hash here

Reply:
Good idea. I have some other logging changes planned and would like to change it there

https://github.com/0xB10C/bademeister-go/issues/35